README.md

```diff
@@ -1 +1,175 @@
-# Hello-Microverse-Project-
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>Hello Microverse Project README File</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+**[your_project__name]** is a simple website that is made out of HTML and CSS. It dicplays the text "Hello Microverse!" on the screen, colored cyan in css.
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML
+- CSS
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]** HTML
+- **[key_feature_2]** CSS
+- **[key_feature_3]** HTML, CSS Linters
```

Comment:
- [ ] You did well 😃 here at adding the key features, but please try to remove the text before the feature for example here you wrote ``` **[key_feature_1]** HTML``` kindly try to make it just ```- HTML``` when you do that it will be great for the professional Readme 😊

Reply:
@osamaashraf6 I have fixed the issues, please take a look.